composer.json

```diff
@@ -2,5 +2,12 @@
     "name": "elife/bus-sdk",
     "description": "eLife Sciences bus SDK",
     "type": "library",
-    "license": "MIT"
+    "license": "MIT",
+    "require": {
```

Comment:
shouldn't it be `require-dev` so that it isn't inherited by packages that depend on this?